cleanupWorker/config/defaults.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/slovak-egov/einvoice/pkg/dbutil"
+	"github.com/slovak-egov/einvoice/pkg/loggerutil"
+)
+
+var devConfig = Configuration{
+	Db: dbutil.Configuration{
+		Host: "localhost",
+		Port: 5432,
+		Name: "einvoice",
+		User: "postgres",
+	},
+	Logger: loggerutil.Configuration{
+		LogLevel: logrus.DebugLevel,
+		Format:   "text",
+	},
+	TestInvoiceExpiration: 7 * 24 * time.Hour,
+	Interval:              1 * time.Hour,
```

Comment:
let's lower these values in `dev` environment. You usually want to test/debug something, so expiration can be maybe `10 minutes` and cron interval `5 minues`?
But still you can adjust it using env variable